---
geometry_msgs/PoseArray tables
bool success
string message